src/.github/workflows/{% if ci == 'GitHub' %}test.yml{% endif %}.jinja

```diff
@@ -51,6 +52,12 @@ jobs:
     runs-on: ubuntu-latest
     container: {% raw %}${{ matrix.container }}{% endraw %}
     name: {% raw %}${{ matrix.name }}{% endraw %}
+    {%- if github_ci_extra_env %}
+    env:
+      {%- for key, value in github_ci_extra_env.items() %}
+      {{ key }}: {{ value|tojson }}
+      {%- endfor %}
+    {%- endif %}
```

Comment:
There is already a `env` key below if there are rebel module groups. So I suspect they will conflict and lead to an invalid yaml?

Reply:
The yaml itself should be valid, but I'm not sure about if it's valid configuration for github actions

Reply:
@theangryangel do you think you can address this issue and combine the two env sections?

Reply:
Yup, sorry I let this sit for a bit, I’ll get it sorted first thing tomorrow before I start any other work. Thanks for the prompt